website/src/components/pageStateSelectors/VariantsSelector.tsx

```diff
@@ -41,11 +41,9 @@ export function VariantsSelector({
                         variantFilterConfig={filterConfig}
                         onVariantFilterChange={(variantFilter) => updateVariantFilter(id, variantFilter)}
                     />
-                    <div className='flex items-center'>
-                        <button className='text-sm' onClick={() => removeVariant(id)}>
-                            Remove
-                        </button>
-                    </div>
+                    <button className='text-sm hover:text-gray-500' onClick={() => removeVariant(id)}>
+                        Remove
+                    </button>
```

Comment:
Maybe this is too small for mobile users. Let's see. I'm not blocking on this here.